test/unit/driver-zookeeper.js

```diff
@@ -32,7 +32,11 @@
         process.env.ZOO_LOG_DIR = process.env.HOME + '/zookeeper/';
 
         // Zookeeper needs time to take-off the ground :(
-        s1.start(['start-foreground', process.env.HOME + '/zookeeper/zoo.cfg']).once('started', done);
+        s1.start(['start-foreground', process.env.HOME + '/zookeeper/zoo.cfg'])
+          .once('started', done)
+          .once('error', function(err) {
```

Comment:
I would rather let this as it previously was instead of bubbling up the error (at least for now, and while the other servers don't bubble either).